tensorflow/c/c_api.h

```diff
@@ -900,6 +900,11 @@ TF_CAPI_EXPORT extern void TF_DeleteImportGraphDefOptions(
 TF_CAPI_EXPORT extern void TF_ImportGraphDefOptionsSetPrefix(
     TF_ImportGraphDefOptions* opts, const char* prefix);
 
+// Set bind device for the nodes in the `graph_def` that will be imported into `graph`.
+// `device` is copied and has no lifetime requirements.
+TF_CAPI_EXPORT extern void TF_ImportGraphDefOptionsSetBindDevice(
```

Comment:
We don't use the term "bind" device in other places, how about calling this `TF_ImportGraphDefOptionsSetDefaultDevice` instead?

(And update comments appropriately)